pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -618,8 +619,40 @@ func GetRegionFromEC2Metadata() (string, error) {
 
 // FromRegion is part of zone.Mapper
 func (s *EbsStorage) FromRegion(ctx context.Context, region string) ([]string, error) {
-	// Fall back to using a static map.
-	return staticRegionToZones(region)
+	ec2Cli, err := newEC2Client(region, s.config)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Could not get EC2 client")
+	}
```

Comment:
This wrap adds nothing of value. The message should identify the context of the failure (in this case the creation of the client in FromRegion).